post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: creating-multi-arch-compute-nodes-ibm-power
-include::_attributes/common-attributes.adoc[]
 [id="creating-multi-arch-compute-nodes-ibm-power"]
-= Creating a cluster with multi-architecture compute machines on {ibm-power-title}
+= Creating a cluster with multi-architecture compute machines on IBM Power
```

Comment:
Is there a reason you're removing the attribute from this heading? If there's no specific reason, the attribute should be used.

Reply:
I noticed that these attributes were not resolved properly in the preview. The issue persisted even after multiple retries, so I replaced the attributes with their actual values.